scripts/mkcloud

```diff
@@ -192,16 +192,19 @@ function error_exit
     exitcode=$1
     message=$2
     if [ -z "$SKIPSUPPORTCONFIG" ] ; then
+        local defaults="-l -I 5000"
+        local sc_cmd="supportconfig ${SUPPORTCONFIG_OPTIONS:-$defaults} | wc"
```

Comment:
Actually no, I was right before.  It works in both cases, because both cases are wrapped inside `ssh`.  Check my quoting :-)